app/screens/phone-auth-screen/phone-validation.tsx

```diff
@@ -221,35 +227,49 @@ export const PhoneValidationScreen: React.FC<PhoneValidationScreenProps> = ({
             variables: { input: { phone, code } },
           })
 
-          sessionToken = data?.userLoginUpgrade?.authToken
+          const success = data?.userLoginUpgrade?.success
+          const sessionToken = data?.userLoginUpgrade?.authToken
+          if (success) {
+            logUpgradeLoginSuccess()
+
+            await Keychain.resetGenericPassword()
```

Comment:
can this call throw an error?

Reply:
I actually was going to remove this call now because we don't need to know whether or not an account is already created since the 'create' mutation is renentrant.